src/main/java/org/symphonyoss/symphony/messageml/elements/Link.java

```diff
@@ -116,6 +117,11 @@ public void validate() throws InvalidInputException {
 
   }
 
+  @Override
+  public void updateBiContext(BiContext biContext){
+    biContext.updateItem(getClass().getSimpleName(), ATTR_HREF);
```

Comment:
Is the "Link" / "Button" part a contract with BI when data is going to be used?
I would perhaps hardcode it then to avoid breaking the analytics if we rename the class (without changing the element's name) (or the name probably already exist as a constant somewhere)

Reply:
I'm going to hardcode it for now. We could then check better on the next implementation (when we are going to do this for all elements) if we want to keep it like this or not. 
Because we could use the MESSAGEML_TAG defined in each element which is for example "button" "date-picker" but then we will have "a" for links, ""p" for paragraphs which I don't know if it's acceptable or not.